peps/pep-0694.rst

```diff
@@ -832,6 +741,24 @@ Besides the standard ``meta`` key, this has the following top level keys:
 The ``message`` and ``source`` strings do not have any specific meaning, and are intended for human
 interpretation to aid in diagnosing underlying issue.
 
+File Upload Mechanisms
+----------------------
+
+File Upload Mechanisms, with the exception of ``http-post-application-octet-stream`` are left as an
+implementation detail specific to each server. Servers **MUST** implement a
+``http-post-application-octet-stream`` mechanism as a fallback if no server specific implementations
+exist.
```

Comment:
As a client maintainer, can we require that all servers must implement `http-post-application-octet-stream`, so that it's possible to develop a client compatible with all servers?

Reply:
I would agree that it isn't perfectly clear. I said "MUST" and then dropped an "if" in, will clarify.